db/migrate/20220928144401_add_user_to_reservation.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserToReservation < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :reservations, :user, index: {algorithm: :concurrently}
```

Comment:
What is this magic 😮  Nice 👀

Reply:
I'm afraid I cannot take credit for this magic, I simply followed strong migrations gem suggestions 😁